.gitignore

```diff
@@ -1,4 +0,0 @@
-_site/
-.sass-cache/
-.jekyll-metadata
-*~
```

Comment:
I checked, and my master branch is ahead because I deleted the radioactive-decay node that I erroneously made on my master branch. Paul helped me compare the upstream with my master branch and they were the same.